pnpm-workspace.yaml

```diff
@@ -2,3 +2,6 @@ packages:
   - apps/*
   - packages/*
   - tools/*
+
+  - '!apps/monoweb-kvittering-backend' # python app
+  - '!apps/monoweb-kvittering-frontend' # does not use any monorepo packages as of now, so we exclude it
```

Comment:
Should be added here, its wrong not to because otherwise it gets its own lock

Reply:
Because then it's effectively not a part of the monorepo whoch kind of defeats the purpose of a monorepo

Reply:
and yep ill remove all the scripts, not needed

Reply:
Hmm sure I was thinking that as its using any of the packages it does not really matter